account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  custom,
+  keccak256,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'keccak256' is defined but never used.
<!-- __reviewdog__:ChA0YTllNTUzMWY0OGMzOGNlEgZyZGpzb24= -->